lisp/init-selectrum.el

```diff
@@ -0,0 +1,52 @@
+;;; init-selectrum.el --- Config for selectrum       -*- lexical-binding: t; -*-
+;;; Commentary:
+;;; Code:
+
+(when (maybe-require-package 'orderless)
+  (setq completion-styles '(orderless)))
```

Comment:
You may also want to install some style dispatcher. I am using the following:

~~~ elisp
  (defun my--orderless-dispatch (pattern _index _total)
    (cond
     ((string-prefix-p "!" pattern) `(orderless-without-literal . ,(substring pattern 1)))
     ((string-suffix-p "=" pattern) `(orderless-literal . ,(substring pattern 0 -1)))
     ((string-suffix-p "~" pattern) `(orderless-flex . ,(substring pattern 0 -1)))))
  (setq completion-styles '(orderless)
                orderless-style-dispatchers '(my--orderless-dispatch)
~~~

~~~
literal=
flex~
!without
~~~

Reply:
Also, @clemera convinced to enable the `!without` syntax by default in orderless. I'll do that soon.

Reply:
@minad Ooh, that's neat. Probably more flexibility than I need, but I can see how it would be useful.